connectors/spark/src/test/java/io/unitycatalog/spark/TableReadWriteTest.java

```diff
@@ -173,6 +173,32 @@ public void testCredentialDelta(String scheme) throws ApiException, IOException
     session.stop();
   }
 
+  @Test
+  public void testCredentialCreateDeltaTable() throws IOException {
+    SparkSession session = createSparkSessionWithCatalogs(SPARK_CATALOG, CATALOG_NAME);
+
+    String loc1 = "s3://test-bucket0" + generateTableLocation(SPARK_CATALOG, DELTA_TABLE);
+    setupDeltaTableLocation(session, loc1, new ArrayList<>(0));
+    String t1 = SPARK_CATALOG + "." + SCHEMA_NAME + "." + DELTA_TABLE;
+    session.sql(String.format("CREATE TABLE %s USING delta LOCATION '%s'", t1, loc1));
+    testTableReadWrite(t1, session);
+
+    String loc2 = "s3://test-bucket1" + generateTableLocation(CATALOG_NAME, DELTA_TABLE);
+    setupDeltaTableLocation(session, loc2, new ArrayList<>(0));
```

Comment:
If the location does have a delta table, then delta should create a brand new delta log there. are we testing that?

Reply:
do you mean empty table? let me add a test